Content.Tests/DMProject/Tests/List/ListAppendSelf.dm

```diff
@@ -0,0 +1,8 @@
+/proc/RunTest()
+	L1 = list(1, 2)
+	L1 += L1
+	ASSERT(L1 = list(1, 2, 1, 2)
```

Comment:
This is missing a closing parenthesis. You're also assigning here, not checking for equality.

`==` wouldn't be correct here either since that checks for reference equality with lists. You want `~=` which checks for equality of the contents.
```suggestion
	ASSERT(L1 ~= list(1, 2, 1, 2))
```